### ISSUE TITLE
No Melodic support

### ISSUE THREAD

**Issue by gstavrinos** [rubedos/viper_win#5]
I am opening this issue here, even though it is an Ubuntu-related issue, since the Viper wiki lives here. 

I am trying to install the cvm-* related packages in Ubuntu Bionic (18.04), but the apt repositories are only available for Ubuntu Xenial (16.04).

Is there a workaround or something I am missing?

Thanks,

George

**Comment by HenrikasRS** [comment]
Hello George, thank you for reaching out!
I've talked to the team engineers about it and they confirmed that we do not have a support for Melodic. So we've added source codes for SDK packages to https://github.com/rubedos/viper_ros/tree/master/sdk and you can try to compile them on your platform. Let me know if this helps.

Thanks, 
Henrikas

**Comment by gstavrinos** [comment]
Thanks for making (part of?) the sdk open source. I was able to successfully compile the package under Ubuntu 18.04 and ROS Melodic. I have made a PR in https://github.com/rubedos/viper_ros/pull/1, that addresses all issues encountered. Although I have not tested it, the PR should fix problems with all versions of ROS due to catkin conflict and missing packages in CMakeLists.txt and package.xml files.

Thanks again, 

George #
